src/Informer/Tags.hpp

```diff
@@ -0,0 +1,21 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <string>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+
+/// \cond
+enum class Verbosity;
+/// \endcond
+
+namespace Tags {
+/// \ingroup LoggingGroup
+/// \brief Tag for putting 'Verbosity' in a DataBox.
```

Comment:
Should the quotes be ticks? And should probably be `::Verbosity` so the doxygen link works correctly :)